src/ui/organisms/filePicker/FilePicker.tsx

```diff
@@ -107,6 +107,7 @@ export const FilePicker: React.FC<FilePickerProps> = ({
         error={isError}
         errorMessage={errorMessage}
         onDropped={handleDropped}
+        size={props.size ?? 'large'}
```

Comment:
It would be better here to destructure to get the `size` outside the props (as you did with the `clearAll` variable), I guess.